cluster-autoscaler/config/autoscaling_options.go

```diff
@@ -121,6 +121,8 @@ type AutoscalingOptions struct {
 	GRPCExpanderCert string
 	// GRPCExpanderURL is the url of the gRPC server when using the gRPC expander
 	GRPCExpanderURL string
+	// IncludeSimilarNodegroupsInExpanderOptions is whether CA will include similar nodegroups in its request to the gRPC expander
+	IncludeSimilarNodegroupsInExpanderOptions bool
```

Comment:
"...InGRPCExpanderOptions"?

Reply:
Yeah maybe that's better, I am a bit worried about the length of the name of the flag 😅 but not sure it matters too much.